pkg/systemd/reporting.jsx

```diff
@@ -397,8 +398,26 @@ const WorkflowRow = ({ message, problemState, reportLinks, label, onReportButton
         <Split hasGutter>
             <SplitItem>{label}</SplitItem>
             <SplitItem isFilled>
-                {problemState === ProblemState.REPORTING && <Spinner size="md" /> }
-                {status}
+                <div style={{
+                    display: "flex",
+                    flexDirection: "row",
+                    justifyContent: "center",
+                    alignItems: "center",
+                    width: "100%",
+                    height: "fit-content",
+                    gap: "0.5rem"
+                }}>
```

Comment:
We can't/shouldn't do inline CSS, please move that to pkg/systemd/reporting.scss and use a class. HTML/JSX should just contain the structural bits.

However, this shouldn't use manual layout/css at all. Please use the [PatternFly Flex component](https://www.patternfly.org/layouts/flex) -- check `git grep -w Flex` for lots of existing examples.

Reply:
Sorry for any inconveniences, I'll get to it. Thanks for the review